### DISCUSSION TITLE
Exciting News! magasin Joins the Digital Public Goods Alliance DPG Registry

### DISCUSSION THREAD

**Discussion by merlos** [unicef/magasin#103] [Announcements]
![magasin-dpga](https://github.com/user-attachments/assets/b6cfd3cb-68c2-424c-8dfb-8a76d21d602d)

Today, we are thrilled to announce that **[magasin](https://magasin.unicef.io)** has been officially added to the [Digital Public Goods Alliance (DPGA) DPG Registry](https://app.digitalpublicgoods.net/a/11177). The DPGA and its registry aim to promote digital public goods to create a more equitable world. Being recognized as a Digital Public Good (DPG) recognizes open source projects that have the potential to tackle global challenges.

**Magasin is a groundbreaking cloud-native open-source end-to-end data platform**. It empowers data analysis teams to unlock valuable insights, driving digital transformation effortlessly. With magasin, data teams can seamlessly extract, analyze, and visualize data from multiple sources. The platform evolves with your organization’s data needs, delivering seamless growth without breaking the bank. Its loosely coupled architecture, composed of mature open-source tools, takes flexibility to another level.

For us, being recognized as a digital public good (DPG) means that magasin adheres to open source principles, [data privacy](https://magasin.unicef.io/implement/data-privacy.html), and software best practices. It ensures that our platform does no harm and helps attain the [Sustainable Development Goals (SDGs)](https://magasin.unicef.io/sdgs.html). This recognition is invaluable as it highlights our commitment to creating [open-source software](https://github.com/unicef/magasin), open data, open AI models, open standards, and open content that contribute to a better world.

The [Digital Public Goods Alliance](https://www.digitalpublicgoods.net) is a multi-stakeholder initiative endorsed by the United Nations Secretary-General. It works to accelerate the attainment of the SDGs in low- and middle-income countries by facilitating the discovery, development, use of, and investment in digital public goods.

**Comment by Bunggie** [comment]
How to donate my all domain and my enterprises property in UNICEF or in World food Program
